pkgs/by-name/gr/grayjay/package.nix

```
@@ -0,0 +1,117 @@
+{
+  buildDotnetModule,
+  fetchgit,
+  dotnetCorePackages,
+  pkgs,
+  buildNpmPackage,
+  lib,
+}:
+let
+  version = "3";
+  src = fetchgit {
+    url = "https://gitlab.futo.org/videostreaming/Grayjay.Desktop";
+    rev = "refs/tags/${version}";
+    hash = "sha256-gI1M/rDTSH16eeknHJfLA2iawoA5iGTKHfwUW4/lvv8=";
+    fetchSubmodules = true;
+    fetchLFS = true;
+  };
+  frontend = buildNpmPackage {
+    name = "grayjay-frontend";
+    inherit version;
+
+    src = "${src}/Grayjay.Desktop.Web";
```

Why not use `sourceRoot`?

```suggestion
    inherit src;

    sourceRoot = "Grayjay.Desktop.Web";
```